build/style.css

```diff
@@ -26,62 +26,47 @@ strong {
 	box-sizing: border-box;
 }
 
-.content ul
```

Comment:
Is there any reason why you removed the `.content ul` and similar styling? While its not used here on this specific example, it would get used whenever any article that uses this template has an unordered list in its content

Reply:
Hey, so I deleted that styling accidentally. I wasn't thinking about how this will eventually be a word press template.